### ISSUE TITLE
Clicking "Stop" or resetting TinyG during "place all" job leads to undefined behaviour

### ISSUE THREAD

**Issue by mrandt** [jkuusama/LitePlacer-ver2#10]
If I click stop in the middle of a place job (or reset TinyG by hitting either limit switch or closing reset switch), the software behaves strange.

At first it appears to be busy for up to a minute - I cannot click anything.

Then I get some errors, but they are not meaningful.

In order to restart placing, I have to close and restart application.

**Comment by thethereza** [comment]
yeah, noticed something like this too but it's hard to replicate to find out what's going on.  the tinyg seems unresponsive.  this will not be easy to track down.
